wallet/addresspool.go

```diff
@@ -350,3 +354,16 @@ func (w *Wallet) GetNewAddressInternal() (dcrutil.Address, error) {
 	}
 	return address, err
 }
+
+// RandomAddress returns a random address. Mainly used for 0-value (unspendable)
+// OP_SSTXCHANGE tagged outputs.
+func (w *Wallet) RandomAddress() (dcrutil.Address, error) {
```

Comment:
Since this has absolutely nothing to do with the wallet, I would make this a non-method function and pass in the chain parameters as a parameter.  I'd also unexport it.